lib/Conversion/LLHDToLLVM/LLHDToLLVM.cpp

```diff
@@ -1884,14 +1884,13 @@ void LLHDToLLVMLoweringPass::runOnOperation() {
 
   LLVMConversionTarget target(getContext());
   target.addIllegalOp<InstOp>();
-  target.addLegalOp<UnrealizedConversionCastOp>();
   cf::populateControlFlowToLLVMConversionPatterns(converter, patterns);
   arith::populateArithToLLVMConversionPatterns(converter, patterns);
 
   // Apply the partial conversion.
   if (failed(
           applyPartialConversion(getOperation(), target, std::move(patterns))))
-    signalPassFailure();
+    return signalPassFailure();
```

Comment:
Started early returning here - doesn't seem like a good idea to keep trying to convert the operations.